packages/core/src/storage/vectorStore/QdrantVectorStore.ts

```diff
@@ -184,9 +184,14 @@ export class QdrantVectorStore implements VectorStore {
     const { points, ids } = await this.buildPoints(embeddingResults);
 
     const batchUpsert = async (points: PointStruct[]) => {
-      await this.db.upsert(this.collectionName, {
-        points: points,
-      });
+      try {
```

Comment:
@EmanuelCampos, we can't merge this PR currently. Would it make sense to add this code to your `QdrantVectorStore`?